templates/kops/kops-private-topology.yaml.gotmpl

```diff
@@ -44,6 +44,12 @@ spec:
     anonymousAuth: false
     authorizationMode: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_MODE" "RBAC" }}
   {{- end }}
+  {{- if bool (getenv "KOPS_KUBE_API_SERVER_FEATURE_GATES_ENABLED" "false") }}
+    featureGates:
+      {{- range $feature := strings.Split " " (getenv "KOPS_KUBE_API_SERVER_ENABLED_FEATURES" "") }}
```

Comment:
The convention is to use comma-separated lists. Also, if we are going to support feature gates as an option, we should to be able to set them to "true" or "false". 

Also, `featureGates` shows up in a bunch of sections:
 - kubeAPIServer
 - kubelet
 - kubeScheduler
 - kubeProxy
 - kubeControllerManager

Some featureGates need to be enabled in more than one section to work. 

So I'd like a more flexible and comprehensive solution. 

@osterman How do you feel about moving the ball forward with partial support?

Reply:
I'd be fine having it be both true and false, and changing the separator to a `,`  I'll figure out a way to do that.

Of note, I think the only reasonable way is to have both an ENABLED and DISABLED variable.  Let me see what I can come up with.

Reply:
Refactored and added in the other managed resources that were already in the file.  did not add new sections for scheduler or controller manager